<launch>
  <node name="iologik" pkg="iologik" type="iologik" output="screen">
    <param name="rate" value="10" />
    <param name="enable_i0" value="false" />
    <param name="enable_i1" value="false" />
    <param name="enable_i2" value="false" />
    <param name="enable_i3" value="false" />
    <param name="enable_i4" value="false" />
    <param name="enable_i5" value="false" />
    <param name="enable_i6" value="false" />
    <param name="enable_i7" value="false" />
    <param name="enable_o0" value="true" />
    <param name="enable_o1" value="true" />
    <param name="minimum_output_current" value="4" />
    <param name="maximum_output_current" value="20" />
    <param name="minimum_input_current" value="4" />
    <param name="maximum_input_current" value="20" />
    <remap from="/input1" to="/iologik/input1" />
    <remap from="/input2" to="/iologik/input2" />
    <remap from="/input3" to="/iologik/input3" />
    <remap from="/input4" to="/iologik/input4" />
    <remap from="/input5" to="/iologik/input5" />
    <remap from="/input6" to="/iologik/input6" />
    <remap from="/input7" to="/iologik/input7" />
    <remap from="/input8" to="/iologik/input8" />
    <remap from="/output1" to="/iologik/output1" />
    <remap from="/output2" to="/iologik/output2" />
  </node>
</launch>